.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea/
+node_modules/
+lib/
+eslint/
```

Comment:
Although the jtest project is not doing this, I believe the dist folder should be ignored since it is generated when packaging. 
It looks like they check in the XSL file into the dist folder too. I think it's not necessary to be like that. 
Let me know if I'm wrong.

Reply:
We can keep it like this for now and figure out when we need to release.